crates/matrix-sdk/src/oidc/mod.rs

```diff
@@ -300,17 +300,17 @@ impl Oidc {
     /// olm machine has been initialized.
     ///
     /// Must be called after `set_session_meta`.
-    async fn deferred_enable_cross_process_refresh_lock(&self) -> Result<()> {
+    async fn deferred_enable_cross_process_refresh_lock(&self) {
         let deferred_init_lock = self.ctx().deferred_cross_process_lock_init.lock().await;
 
         // Don't `take()` the value, so that subsequent calls to
         // `enable_cross_process_refresh_lock` will keep on failing if we've enabled the
         // lock at least once.
         let Some(lock_value) = deferred_init_lock.as_ref() else {
-            return Ok(());
+            return;
         };
 
-        // FIXME We shouldn't be using the crypto store for that! see also https://github.com/matrix-org/matrix-rust-sdk/issues/2472
+        // FIXME: We shouldn't be using the crypto store for that! see also https://github.com/matrix-org/matrix-rust-sdk/issues/2472
```

Comment:
:monocle_face:

Reply:
You're still not using [trouble](https://github.com/folke/trouble.nvim) to highlight and quickly list those?

![image](https://github.com/matrix-org/matrix-rust-sdk/assets/552026/de1011f9-e603-452c-8fda-087e879d2bab)

Reply:
uhuh i'm getting into trouble right now!

Reply:
:drum: